internal/app/api/v1/scripts.go

```diff
@@ -158,11 +159,12 @@ func (s kubtestAPI) ExecuteScript() fiber.Handler {
 			Params:     request.Params,
 		}
 		s.Log.Infow("calling executor with options", "options", options)
-		execution, err := executor.Execute(options)
+		// execution, err := executor.Execute(options)
```

Comment:
I would leave HTTP based executors and implement both methods - it's quite nice feature for possible integrations - not all integrations could be fat like Cypress.

Reply:
Sounds good we only need to figure out a way how to do that. Which also opens the question if we are going to need typed executors anymore?

Reply:
how will you know then which agent to run?

Reply:
Based on CR, we have field Type_ which should be enough to make the call.